noxfile.py

```diff
@@ -3,5 +3,5 @@
 
 @nox.session
 def test(session: nox.Session) -> None:
-    session.install(".", "pytest", "build", "meson-python", "ninja")
+    session.install(".", "pytest", "build", "meson-python", "ninja", "gcovr")
```

Comment:
@stefanv / @jarrodmillman , should this be part of a `test_requirements.yaml` for depandabot to pick up and auto update or this is ok?